### ISSUE TITLE
raising an error as part of a boolean expression crashes

### ISSUE THREAD

**Issue by Licenser** [alpaca-lang/alpaca#183]
```
false && (error \"oh no zero!\")
```

this crashes with:

```
** exception error: no case clause matching {raise_error,1,error,{string,1,"oh no zero!"}}
     in function  alpaca_ast_gen:term_line/1 (/Users/heinz/Projects/Other/alpaca/_build/shell/lib/alpaca/src/alpaca_ast_gen.erl, line 290)
     in call from alpaca_parser:yeccpars2_138/7 (/Users/heinz/Projects/Other/alpaca/_build/test/lib/alpaca/src/alpaca_parser.yrl, line 487)
     in call from alpaca_parser:yeccpars0/5 (/usr/local/Cellar/erlang/19.3/lib/erlang/lib/parsetools-2.1.4/include/yeccpre.hrl, line 57)
```

but should probably generate a Line number

**Comment by Licenser** [comment]
This is fixed as part of #182

**Comment by j14159** [comment]
Closing because of PR #182
